package.json

```diff
@@ -43,7 +43,7 @@
 		"@godaddy/terminus": "^4.11.0",
 		"@graphql-tools/load": "^7.7.0",
 		"@graphql-tools/url-loader": "^7.17.18",
-		"@kiva/kv-components": "^6.69.3",
+		"@kiva/kv-components": "^7.1.0",
```

Comment:
Looks like your lock file has a version change. Please try reverting both of these files and regenerating with what I'm assuming is a lower node version. Use `nvm use` in the repo folder first. I'm personally using node `v22.16.0` and npm `10.9.2`